recipe/meta.yaml

```diff
@@ -29,7 +30,6 @@ requirements:
     - json5 >=0.9.0
     - jsonschema >=4.17.3
     - jupyter_server >=1.21,<3
-    - packaging >=21.3
```

Comment:
packaging >=21.3 is still required, see https://github.com/jupyterlab/jupyterlab_server/blob/v2.22.0/pyproject.toml#L36

Reply:
I'll suppress the linter then